source/concepts/networking/network-firewall.html.md.erb

```diff
@@ -0,0 +1,60 @@
+---
+owner_slack: "#modernisation-platform"
+title: AWS Network Firewall
+last_reviewed_on: 2023-02-08
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+We currently have setup in our core-network account AWS Network Firewall to monitor and control network traffic on a layer 3-7 level for transit gateway connections and cross account VPCs to protect resources
+such as EC2 instances, along with a general AWS resources
+
+### What is an AWS network firewall
+
+To learn more about AWS firewall please follow the links bellow
+
+- [AWS Firewall](https://aws.amazon.com/network-firewall/features/)
+- [AWS Firewall Training](https://www.youtube.com/watch?v=WNFknf9zyZg)
+
+### Amending Rules
+
+Rules can be amended by creating a pull request against the modernisation-platform repo and amending one of the following four file names from the core-network-services in the terrafrom environments folder.
```

Comment:
`in the terraform/environments/core-network-services directory` rather than `from the core-network-services in the terrafrom environments folder`

Reply:
Changed